controller/src/main/java/com/softserve/borda/controllers/TicketController.java

```diff
@@ -264,11 +267,11 @@
         }
     }
 
-    @PostMapping("/tickets/{ticketId}/addMember/{userId}")
+    @PostMapping("/tickets/{ticketId}/addMember")
```

Comment:
Bad resource naming.
Please read https://restfulapi.net/resource-naming/ doc and change naming.

Reply:
Fixed in FixResourcesPaths branch/ticket, already merged in main